### PR TITLE
Correct typo missing by cspell on certain file type

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
`cspell` was missing some file type because of `type_or` by default it looks for all `text` file so it's enough.

### CODE REVIEW

File: parsec/core/fs/workspacefs/workspacefile.py [hunk 1]
```diff
@@ -33,13 +33,13 @@ class WorkspaceFile:
     Keyword arguments:
     transactions -- object to use for file transactions.
     path -- relative path of the file.
-    mode -- the mode where the file have been opened. It needs to have exactly one of "awrx".
+    mode -- the mode where the file have been opened. It needs to have exactly one of "arwx".
```
Comment (Volham22) [inline parsec/core/fs/workspacefs/workspacefile.py:36]: Is it a false positive? I think here `r`, `w` and `x` stands for read, write and execute.

File: parsec/core/fs/workspacefs/workspacefile.py [hunk 2]
```diff
@@ -33,13 +33,13 @@ class WorkspaceFile:
...
     mode("a") -> append file.
-    mode("b") -> have to be used with one of "awrx" mode, file will be write/read as bytes instead of string.
+    mode("b") -> have to be used with one of "arwx" mode, file will be write/read as bytes instead of string.
```
Comment (Volham22) [inline parsec/core/fs/workspacefs/workspacefile.py:38]: Same problem here

File: parsec/core/fs/workspacefs/workspacefile.py [hunk 3]
```diff
@@ -33,13 +33,13 @@ class WorkspaceFile:
...
     mode("w") -> write file. If the file doesn't exist, create one.
     mode("r") -> read file.
     mode("x") -> If the file doesn't exist, create one, else, raise an Error. File will be opened in write mode.
-    mode("+") -> have to be used with one of "awrx" mode, file will be in read/write mode.
+    mode("+") -> have to be used with one of "arwx" mode, file will be in read/write mode.
```
Comment (Volham22) [inline parsec/core/fs/workspacefs/workspacefile.py:42]: and here too.

File: tests/common/postgresql.py [hunk 4]
```diff
@@ -50,7 +50,7 @@ def bootstrap_postgresql_testbed():
         _pg_db_url = _patch_url_if_xdist(provided_db)
 
     print("PostgreSQL url: ", _pg_db_url)
-    # Finally initialiaze the database
+    # Finally initialaze the database
```
Comment (Volham22) [inline tests/common/postgresql.py:53]: `initialize` ?

**Review by ADcorpo — APPROVED** [review]
looks good to me !

**Review by Volham22 — CHANGES_REQUESTED** [review]

File: parsec/core/fs/workspacefs/workspacefile.py (hunk 1 above)
Comment (FirelightFlagboy) [reply parsec/core/fs/workspacefs/workspacefile.py:36]: Yes it is, it's just the a prefer the word order `read write execute`

File: parsec/core/fs/workspacefs/workspacefile.py (hunk 2 above)
Comment (FirelightFlagboy) [reply parsec/core/fs/workspacefs/workspacefile.py:38]: So we continue in https://github.com/Scille/parsec-cloud/pull/3772#discussion_r1049637877

File: parsec/core/fs/workspacefs/workspacefile.py (hunk 3 above)
Comment (FirelightFlagboy) [reply parsec/core/fs/workspacefs/workspacefile.py:42]: https://github.com/Scille/parsec-cloud/pull/3772#discussion_r1049637877

File: tests/common/postgresql.py (hunk 4 above)
Comment (FirelightFlagboy) [reply tests/common/postgresql.py:53]: yes

Comment (FirelightFlagboy) [reply tests/common/postgresql.py]: done

**Review by Volham22 — APPROVED** [review]

**Pull request merged**